sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/HttpHelperTests.cs

```diff
@@ -183,6 +183,131 @@ public void GetUrl_HttpUrl_Success()
             Assert.Equal("www.wiki.com", urlAuthority);
         }
 
-        // TODO: Order of precedence.
+        [Fact]
+        public void HttpRequestUrlIsSetUsingHttpUrl()
+        {
+            var PartBTags = AzMonList.Initialize();
+            AzMonList.Add(ref PartBTags, new KeyValuePair<string, object>(SemanticConventions.AttributeHttpUrl, "https://www.wiki.com"));
+
+            string url = PartBTags.GetRequestUrl();
+            Assert.Equal("https://www.wiki.com", url);
+        }
+
+        [Fact]
+        public void HttpRequestUrlIsSetUsing_Scheme_Host_Target()
+        {
+            var PartBTags = AzMonList.Initialize();
+            AzMonList.Add(ref PartBTags, new KeyValuePair<string, object>(SemanticConventions.AttributeHttpScheme, "http"));
+            AzMonList.Add(ref PartBTags, new KeyValuePair<string, object>(SemanticConventions.AttributeHttpHost, "www.httphost.org"));
+            AzMonList.Add(ref PartBTags, new KeyValuePair<string, object>(SemanticConventions.AttributeHttpTarget, "/path"));
+
+            string expectedUrl = "http://www.httphost.org/path";
+            string url = PartBTags.GetRequestUrl();
+            Assert.Equal(expectedUrl, url);
+        }
+
+        [Theory]
+        [InlineData("80")]
+        [InlineData("443")]
+        [InlineData("8888")]
```

Comment:
Looks like port is a `long` type in OpenTelemetry SDK. Please validate, not a blocker for this PR.

Reply:
Defined as int in [spec](: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/span-general.md#general-network-connection-attributes)
Will add this to do list and update all tets